pkgs/development/python-modules/scikitlearn/default.nix

```
@@ -40,6 +41,15 @@ buildPythonPackage rec {
   ];
   checkInputs = [ pytest ];
 
+  patches = [
+    # Fixes tests by changing threshold of a test-case that broke
+    # with numpy versions >= 1.17. This should be removed for versions > 0.21.2.
+	( fetchpatch {
+	  url = "https://github.com/scikit-learn/scikit-learn/commit/b730befc821caec5b984d9ff3aa7bc4bd7f4d9bb.patch";
+	  sha256 = "0i621flcfqgjcw37p96zx7hfrkghlvd88wi816i4kbs0gxg0vm6p";
```

I'm getting a different hash
```suggestion
	  sha256 = "0z36m05mv6d494qwq0688rgwa7c4bbnm5s2rcjlrp29fwn3fy1bv";
```

---

Hm, let me double check. Likely I messed it up locally.

---

Fixed, thanks.